website/project/model.py

```diff
@@ -3484,7 +3484,7 @@ def terminate_embargo(self, auth):
             action=NodeLog.EMBARGO_TERMINATED,
             params={
                 'project': self._id,
-                'node': self._id,
+                'node': self.registered_from_id,
```

Comment:
Based on review from Product, revert this line and don't worry about doing a migration.